pyquil/paulis.py

```diff
@@ -286,14 +298,7 @@ def __rsub__(self, other):
         return other + -1. * self
 
     def __str__(self):
-        term_strs = []
-        for index in self._ops.keys():
-            term_strs.append("%s%s" % (self[index], index))
-
-        if len(term_strs) == 0:
-            term_strs.append("I")
-        out = "%s*%s" % (self.coefficient, '*'.join(term_strs))
-        return out
+        return f'{self.coefficient}*{self.id(sort_ops=False)}'
```

Comment:
Seems like a breaking change to change the string representation of an object. Do we really want to do this?

Reply:
oh, I'll put this back

Reply:
got confused because `term.id()` and `term.pauli_str` just jam them all together. It's back to how it was and I added a new method `compact_str` because I like it better without all the `*` and it's easier to parse back out